components/api/api.tsx

```diff
@@ -48,3 +50,31 @@ export async function getArticleComment(articleId: number, params = { limit: 10,
         throw error;
     }
 }
+export async function postSignUp(signUpData : SignUpFormInputs) {
+    try {
+        const response = await axios.post(`${BASE_URL}/auth/signUp`, {
+            email:signUpData.email,
+            nickname:signUpData.nickname,
+            password:signUpData.password,
+            passwordConfirmation:signUpData.passwordConfirm,
+        });
```

Comment:
P3:
아래 코드도 동일동작합니다~
signUpData 매개변수 객체 자체를 쓰는 일이 없으니 아래처럼 받아서 사용하시는게 가독성 측면에서 더 좋을 것 같습니다~

```suggestion
export async function postSignUp({
  email,
  nickname,
  password,
  passwordConfirm: passwordConfirmation,
}: SignUpFormInputs) {
  try {
    const response = await axios.post(`${BASE_URL}/auth/signUp`, {
      email,
      nickname,
      password,
      passwordConfirmation,
    });

```